src/index.ts

```diff
@@ -20,8 +20,10 @@ function doCallback() {
   });
 }
 
+export type Subscribeable = Pick<Replicache, 'subscribe'>;
```

Comment:
What I was thinking was actually describing the subscribe interface. But now that I see maybe why you didn't want to do that - the subscribe interface is large and detailed. It could also fall behind head of either Replicache or Reflect.

What if instead we have this package just directly depend on Replicache and Reflect and have the argument be `Replicache`|`Reflect`.

I think that is fine, it's not going to add to the compile size, or even the install size for our users -- it just means when we work on this we install both packages.